sympy/solvers/ode/single.py

```diff
@@ -791,66 +791,93 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [gensol]
 
 
-class SecondNonlinearAutonomousConserved(SinglePatternODESolver):
+class SecondAutonomousNth(SinglePatternODESolver):
     r"""
-    Gives solution for the autonomous second order nonlinear
-    differential equation of the form
+    The general second order autonomous differential equation of the form:
+    .. math :: x''(t) + f(x)*x'(t) + g(x) = 0
```

Comment:
A general 2nd order autonomous ODE would be:
```
x''(t) = g(x'(t), x(t))
```

Reply:
There should be a blank line before the equation

Reply:
Sorry, I will correct it.